__init__.py

```diff
@@ -1,16 +1,15 @@
-from adapt.intent import IntentBuilder
-from mycroft import MycroftSkill, intent_handler
-from ovos_utils.process_utils import RuntimeRequirements
 from ovos_utils import classproperty
+from ovos_utils.intents import IntentBuilder
+from ovos_utils.process_utils import RuntimeRequirements
+from ovos_workshop.decorators import intent_handler
+from ovos_workshop.skills import OVOSSkill
 
 
-class HelloWorldSkill(MycroftSkill):
-    def __init__(self):
+class HelloWorldSkill(OVOSSkill):
+    def __init__(self, *args, **kwargs):
         """ The __init__ method is called when the Skill is first constructed.
-        It is often used to declare variables or perform setup actions, however
-        it cannot utilise MycroftSkill methods as the class does not yet exist.
         """
```

Comment:
perhaps note that self.bus, self.skill_id, self.settings etc are only available after the call to super()

since this might be a new concept for newbies let's be verbose